### PR TITLE
feat(agora): migration Home component (AG-1547)

### PR DESCRIPTION
**Pull request by sagely1**
## Description

Migration of Home component

Note: Header and Footer will be updated in a future PR.  Also, Gene search functionality has been commented out for now but will be implemented as part of the header as they share this functionality.

## Related Issue

https://sagebionetworks.jira.com/browse/AG-1546

## Changelog

- Add home component
   
## Testing
`nx test agora-home`

## Expected output
Navigate to:
http://localhost:4200

image

### CODE REVIEW

File: libs/agora/home/.hintrc
Comment (tschaffter) [inline libs/agora/home/.hintrc:1]: This file should be removed now that webhint (HTML linter) has been removed.

File: libs/agora/home/project.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      }
+    },
+    "lint-html": {
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": ["hint src/**/*.html"],
+        "cwd": "libs/agora/home",
+        "parallel": false
+      }
+    }
```
Comment (tschaffter) [inline libs/agora/home/project.json:31]: Remove this task now that webhint has been removed. HTML files are now linted as part of the `lint` task.

**Review by tschaffter — CHANGES_REQUESTED** [review]

File: libs/agora/home/.hintrc
Comment (sagely1) [reply libs/agora/home/.hintrc:1]: done!

File: libs/agora/home/project.json (hunk 1 above)
Comment (sagely1) [reply libs/agora/home/project.json]: done!

**Review by tschaffter — APPROVED** [review]

**Pull request merged**